server/src/completions.ts

```diff
@@ -364,68 +401,115 @@ function getFieldsFromCurrentBlock(
     if (key === block.end.line) {
       break
     }
-    if (key !== position.line) {
+    if (!position) {
+      suggestions.push(item.replace(/ .*/, ''))
```

Comment:
`item.trimLeft()`

Reply:
Want I actually want to do here is get the first word in the line, not trim it from from side.